cookbook/integrations/kubernetes/k8s_spark/Dockerfile

```diff
@@ -8,7 +8,8 @@ ENV LC_ALL C.UTF-8
 ENV PYTHONPATH /root
 ENV DEBIAN_FRONTEND=noninteractive
 
-# Install Python3 and other basics
+## Install Python3 and other basics
+USER 0
```

Comment:
Does it work with a non-root user? depending on what it does you may be ok running `USER 0` here but then setting it to `USER 1001` (or something) at line 26 (doing the python stuff)... We did a pass before to make all of our flytesnacks images run as non-root to be secure by default and encourage best practices and want to keep that going